htm.it/conf/application.tpl

```diff
@@ -56,6 +56,13 @@ poll_interval = 60
 # ERROR state if it continues to encounter errors
 metric_error_grace_period = 10800
 
+[metric_poller]
+graphite_url = https://graphite-dfw.pardot.com
+graphite_username =
+graphite_password =
+poll_frequency = 5
```

Comment:
I'm guessing that 5 will be too low for a production value, 15-60 is more likely to be doable, but we can leave this and return once we have it live.